### ISSUE TITLE
esx_identity error for currentIdentity

### ISSUE THREAD

**Issue by madhatter23412** [FiveEYZ/esx_kashacter#23]
Running ES_Extended 1.2 
All Updated
Everything Runs, Saves and Loads Characters Right with no Issues. 
However, everytime someone loads into server I get an 
SCRIPT ERROR: @esx_identity/server/main.lua:198: attempt to index a nil value (local 'currentIdentity')
and Players Load in as Micheal. 
Is there a step I missed or something I need to change? 
I increased the Char limit on everything even the firstname and lastname.

**Comment by madhatter23412** [comment]
Solved 
Esx_Identity 1.4.0
SCRIPT ERROR: @esx_identity/server/main.lua:198: attempt to index a nil value (local 'currentIdentity')
Spawning as Michael
In esx_identity/server/main.lua 
find  line 134

131	else
132			for k,v in ipairs(GetPlayerIdentifiers(playerId)) do
133				if string.match(v, 'license:') then
134					identifier = string.sub(v, 9) <------ should say identifier = v
					break
				end
			end
		end

Basically the same step you do in es_extended. for onPlayerJoined(playerId)
